pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java

```diff
@@ -55,7 +55,7 @@
         }, description = "Local zookeeper session timeout ms")
         private int zkSessionTimeoutMillis = 30000;
 
-        @Parameter(names = { "-c", "-cluster" }, description = "Cluster name")
+        @Parameter(names = { "-c", "--cluster" }, description = "Cluster name")
```

Comment:
I would suggest keeping the old way in case there already has people used to deploy their cluster. We can just add the new name to it.

Reply:
Got, so add the new `--cluster`